tiledb/bioimg/converters/base.py

```diff
@@ -213,15 +213,17 @@ def to_tiledb(
                 chunked downsampling. If None, it will default to the number of processors
                 on the machine, multiplied by 5.
         """
-        if cls._ImageReaderType is None:
+        if isinstance(input_path, ImageReader):
```

Comment:
In case of misuse doesn't this allow a Frankenstein expression of `OmeZarrConverter.to_tiledb(OmeTiffReader(...),...)` ?

Reply:
Good point, currently this works and just ignores the class. It is possible to detect this case and raise an error, if desired.